src/main/content/_assets/js/openliberty.js

```diff
@@ -70,24 +70,18 @@ var openliberty = (function() {
     // slide nav bar into view, move down elements that are fixed to top of screen
     function showNav() {
         var nav_height = $("#nav_bar").outerHeight();
-        var accord_height = $("#mobile_toc_accordion").outerHeight();
+
         // fix nav bar to top of screen
         $("#nav_bar").addClass("fixed_top");
         $("#nav_bar").removeClass("hide_nav");
 
         // push toc column, toc indicator and code column down below nav bar
-        $("#toc_column").css("top",accord_height + nav_height + "px");
+        $("#toc_column").css("top", nav_height + "px");
         $("#toc_indicator").css("margin-top", nav_height + "px");
 
         // add margin-top to body so page doesn't jump when nav slides into view
         $('body').css("margin-top", nav_height + "px");
 
-        // To set the toc column to the top when mobile toc accordion returns to original position
-        var mobile = $('#mobile_toc_accordion_container');
-        if(!mobile.classList.contains("fixed_toc_accordion")){
```

Comment:
This PR needed to be reverted from draft because `contains()` is not a valid method in this code.  Please either use the JS `includes()` method or the JQuery `hasClass()` method.